docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -353,8 +353,7 @@ Contributors are encouraged to run coverage on the test suite to identify areas
 that need additional tests. The coverage tool installation and use is described
 in :ref:`testing code coverage<topics-testing-code-coverage>`.
 
-Coverage should be run in a single process to obtain accurate statistics. To
-run coverage on the Django test suite using the standard test settings:
+To run coverage on the Django test suite using the standard test settings:
```

Comment:
I'm not sure why we want to change this without changing commands :thinking:

Reply:
I agree. We could document the single and multiple processes options.